README.md

```diff
@@ -220,9 +316,9 @@ curl --location --request POST 'https://localhost:8083/proxy' \
     "requestedArtifact" : "test1.csv"
```

Comment:
Is this one just test1.csv or it needs to be in URI format?

Reply:
You're right, changed!